site/management.md

```diff
@@ -395,6 +402,22 @@ For instance, if you configured the CSP header with the value `default-src 'self
 
 In addition to the `connect-src` CSP header, RabbitMQ also needs the CSP directives `unsafe-eval` `unsafe-inline`, otherwise the OAuth 2.0 functionality may not work.
 
+### Identity-Provider initiated logon
+
+By default, the Management UI uses OAuth 2.0 authorization code flow to authenticate and authorize users.
+However, there are scenarios where users preferred to be automatically redirected to RabbitMQ without getting
+involved in additional logon flows. This is common in Web Portals where with a single click, users navigate
+straight to a RabbitMQ cluster's management UI with a token obtained under the covers. This is known as
+**Identity-Provider initiated logon**.
+
+RabbitMQ exposes a new setting called `management.oauth_initiated_logon_type` whose default value `sp_initiated`.
+To enable an **Identity-Provider initiated logon** we set it to `idp_initiated`.
+
+When we set `management.oauth_initiated_logon_type` to `idp_initiated` the minimum required configuration is
+`oauth_enabled: true` and `oauth_provider_url`. The other settings related to OAuth are not required.
+The `oauth_provider_url` should be the web portal address.
+
+Once we set `management.oauth_initiated_logon_type` to `idp_initiated`, the management UI exposes the endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field.
 
```

Comment:
Should the above be updated to: "After you set `management.oauth_initiated_logon_type` to `idp_initiated` and `oauth_enabled: true` and o`auth_provider_url` are configured, the management UI exposes the endpoint `/login` which accepts `content-type: application/x-www-form-urlencoded` and it expects the JWT token in the `access_token` form field.